.github/workflows/pr-create-release-build.yml

```diff
@@ -0,0 +1,207 @@
+      pr_number: ${{ steps.create-pull-request.outputs.pr_number }} 
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+          fetch-depth: 0
+
+      - name: Set up Git
+        run: |
+          git config --global user.name 'github-actions'
+          git config --global user.email 'github-actions@github.com'
+      - name: Create and checkout new branch
+        id: create_branch
+        run: |
+          BRANCH_NAME="update-param-env-${{ github.event.inputs.tag_name }}"
+          echo "BRANCH_NAME=${BRANCH_NAME}" >> $GITHUB_ENV
+          git checkout -b $BRANCH_NAME
+      - name: Update params.env with new release version
+        run: |     
+          sed -i 's|:v[0-9.]*\b|:${{ github.event.inputs.tag_name }}|gm' config/overlays/odh/params.env
+      - name: Commit changes
+        run: |
+          git add config/overlays/odh/params.env
+          git commit -m "Update image refs for odh release"
+          git push origin $BRANCH_NAME
+      - name: Create Pull Request
+        id: create-pull-request
+        run: | 
+          PR_URL=$(gh pr create -B ${{ github.ref }} -H ${{ env.BRANCH_NAME }} --title '[ODH Release] Update images for ${{ github.event.inputs.tag_name }}' --body 'Update images for ${{ github.event.inputs.tag_name }}')
+          echo "PR_URL=${PR_URL}" >> $GITHUB_ENV
+          pr_number=$(echo "$PR_URL" | grep -o -E '[0-9]+$')
+          echo "pr_number=${pr_number}" >> $GITHUB_OUTPUT
+        env:
+          GH_TOKEN: ${{ github.token }}
+
+  wait-checks:
+    runs-on: ubuntu-latest
+    needs: [ check-prev-tag,create-pr ] 
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+
+      - name: Watching PR if Checks finished without errors
+        id: wait-checks
+        run:
+          gh pr checks  ${{ needs.create-pr.outputs.pr_number }}  --watch --fail-fast 
+        env:     
+         GH_TOKEN: ${{ github.token }}
+  comment-lgtm:
+    needs: [ check-prev-tag,create-pr,wait-checks ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.ref }}
+      
+      - name: Add comment to trigger lgtm label
+        if: ${{ needs.wait-checks.result == 'success' }}
+        run: |
+          gh pr comment ${{ needs.create-pr.outputs.pr_number }} --body "/lgtm"
+        env:
+          GH_TOKEN: ${{ github.token }}
+
+  rename-image:
+    needs: [ check-prev-tag,create-pr,wait-checks,comment-lgtm]
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Log in to Quay.io
+        uses: docker/login-action@v2
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.Q_PASSWORD }}
+    
+      - name: Wait for PR to be merged

```

Comment:
For the PR to be merged by openshift-ci, it needs to be both /approved and /lgtm.
Not anybody commenting with `/lgtm` will lead to the PR being labeled as "LGTM". Only people listed in the owners file can give /approve and /lgtm: https://github.com/opendatahub-io/kserve/blob/master/OWNERS.

The PR is going to be created by GitHub actions, so it won't have approved status. Also, openshift-ci won't let you approve your own PRs... So, I don't think this will work...

Reply:
So i guess it is better to remove this part as /approve or /lgtm can be commented only by the owners.